tests/manage/configuration/test_configuration.py

```diff
@@ -41,6 +41,11 @@ def test_check_version_development(monkeypatch, capsys, isolated_config, suppres
 
     isolated_config.set_option('warnings.development_version', not suppress_warning)
 
+    # The following is a workaround for the fact that the default loaded profile that comes with the ``isolated_config``
+    # fixture comes with ``warnings.development_version`` set to ``False``. This value will override the value that was
+    # set globally on the config above, and will cause the test to fail.
+    aiida_profile.unset_option('warnings.development_version')
```

Comment:
Can this `unset_option` happened in `isolated_config` fixture to have a consistent isolated_config? I don't think it is clear to have two profiles involved here.

Reply:
Nice, it makes it easier to understand then.